wire-ios-data-model/Source/ManagedObjectContext/NSPersistentStore+Metadata.swift

```diff
@@ -38,7 +38,7 @@ extension Int: SwiftPersistableInMetadata {}
 // swiftlint:disable:next todo_requires_jira_link
 // TODO: Swift 4
 // extension Array where Element == SwiftPersistableInMetadata: SwiftPersistableInMetadata {}
-public extension NSManagedObjectContext {
+public extension NSManagedObjectContext { // TODO: needed on import?
```

Comment:
I guess yes, what's your question exactly @caldrian ?

Reply:
Sorry, I wanted to remove this TODO before you start reviewing. Already deleted now.